scripts/hail_batch/variant_selection/hgdp_1kg_tob_wgs_variant_selection.py

```diff
@@ -11,7 +11,7 @@
 
 TOB_WGS = 'gs://cpg-tob-wgs-main/mt/v2-raw.mt/'
 
-NUM_ROWS_BEFORE_LD_PRUNE = 1000000
+NUM_ROWS_BEFORE_LD_PRUNE = 500000
```

Comment:
I'd suggest being a bit more aggressive in downsizing here -- you can always re-run this in case we get too few loci in the end. (That's more rewarding than no output ;).)

```suggestion
NUM_ROWS_BEFORE_LD_PRUNE = 200000
```

Reply:
"That's more rewarding than no output" - 100% agree. I've changed this to 200k